tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Hooks/IntegrationHook.cs

```diff
@@ -13,10 +14,13 @@ namespace NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Hooks
     public sealed class IntegrationHook
     {
         private readonly IObjectContainer _objectContainer;
+        private readonly DatabaseHealthCheck _databaseHealthCheck;
```

Comment:
The name of the class `DatabaseHealthCheck` is a little misleading here as we are checking the health endpoints of the Ordering API which just so happens to check the database. If we added a new dependency to the Ordering Api this might add more confusion here. Off the top of my head for a better name, how about `OrderingApiHealthCheck`?

Reply:
👍